api/src/schemas.py

```diff
@@ -201,10 +203,17 @@ class S3Input(WorkflowInputBase):
     end_datetime: Optional[datetime]
     single_datetime: Optional[datetime]
 
+    class Config:
+        extra = Extra.allow
+
     @root_validator
-    def is_accessible(cls, values):
-        bucket, prefix = values.get("bucket"), values.get("prefix")
-        validators.s3_bucket_object_is_accessible(bucket=bucket, prefix=prefix)
+    def object_is_accessible(cls, values):
+        bucket = values.get("bucket")
+        prefix = values.get("prefix")
+        zarr_store = values.get("zarr_store")
+        validators.s3_bucket_object_is_accessible(
+            bucket=bucket, prefix=prefix, zarr_store=zarr_store
+        )
```

Comment:
Can we add `zarr_store: Optional[str]` to the model definition, and remove the `extra` config option?